locksmith/src/routes/index.ts

```diff
@@ -52,6 +52,17 @@ router.use('/', lockRouter)
 router.use('/users', userRouter)
 router.use('/purchase', purchaseRouter)
 router.use('/claim', claimRouter)
+
+// Redirects for broken base URLs
+router.use(
+  '/api/key/:chain([0-9]{1,12}):lock(0x[0-9A-Fa-f]{40}):rest(/[0-9]{0,12})?',
```

Comment:
```suggestion
  '/api/key/:chain([0-9]{1,12}):lock(0x[0-9A-Fa-f]{40}):rest(/[0-9]{0,12})?',
```
I wonder, if you do not need to escape `/` in the rest here. Does it work without it?

Reply:
Yes, I tested without `/`, with just `/` and with `/213`... all work!